client/src/components/main/coffee_shop/CoffeeShopPanel.js

```diff
@@ -23,27 +23,28 @@ export default ({ shop, extraContent }) => {
             </span>
           </div>
           <div className="panel-coffee">
+            <span id="panel-coffee">COFFEE</span>
```

Comment:
Here too, because this is being rendered using a `map` function this will result in many objects with the `id` of `panel-coffee` which can cause DOM problems

Reply:
Thaaat makes sense. Corrected now.

Reply:
Thanks John!